.setup/ansible/roles/submitty_course_add_instructor/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+submitty_course_add_instructor_username: instructor
```

Comment:
Is the intention of this to be developer defaults? If not it seems like a bad idea to define these incase they accidentally become used in production. 

It might be a better idea to leave them undefined and if the user does not define or enter them have ansible crash out. Or explicitly give hints they are required and why https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_filters.html#forcing-variables-to-be-defined

Reply:
We removed our default values and instead, created a README that instructed the user to set the desired variables as they wish. If not, the script will produce an error prompting them to set the variables in the playbooks.